service/storage_service.cc

```diff
@@ -3834,11 +3834,6 @@ future<> storage_service::node_ops_abort(node_ops_id ops_uuid) {
                 as->request_abort();
             }
         }
-
-        for (auto it = _node_ops.begin(); it != _node_ops.end(); it = _node_ops.erase(it)) {
```

Comment:
This is mutating _node_ops. I don't think you should remove this code in a patch that just removes unused captures, an explanation is needed why this is safe (if it is safe).

Reply:
ahh, this was a terrible mistake. sorry! fixed in a better way.